src/containers/SubmitSteps/Questionary/Step1d/Step1d.tsx

```diff
@@ -0,0 +1,199 @@
+      hasPcr: state['submit-steps']?.testTaken?.includes('pcr') ?? false,
+      hasAntigen: state['submit-steps']?.testTaken?.includes('antigen') ?? false,
+      // hasAntibody: state['submit-steps'].testTaken.includes('antibody'),
+      country,
+    },
+    resolver: yupResolver(schemaWithoutPatient),
+  });
+  const { errors, isValid } = formState;
+
+  // Callbacks
+  const handleDoBack = React.useCallback(() => {
+    setActiveStep(false);
+    if (previousStep) {
+      history.push(previousStep);
+    } else {
+      history.goBack();
+    }
+  }, [history, previousStep]);
+
+  // Effects
+  useEffect(() => {
+    scrollToTop();
+    if (patientId) {
+      setTitle('');
+    } else {
+      setTitle(t('questionary:resultPcrTest.title'));
+    }
+    if (patientId) {
+      setType('tertiary');
+    } else {
+      setType('primary');
+    }
+    setSubtitle('');
+    setDoGoBack(() => handleDoBack);
+  }, [handleDoBack, setDoGoBack, setTitle, setType, setSubtitle, patientId, t]);
+
+  // Handlers
+  const onSubmit = async (values: Step1dType) => {
+    if (values) {
+      const {
+        pcrTestDate,
+        pcrTestResult,
+      } = (values as any);
+      // if patient
+      if (!pcrTestDate || !pcrTestResult) {
+        return;
+      }
+
+      action(values);
+      if (nextStep) {
+        setActiveStep(false);
+        history.push(nextStep);
+      }
+    }
+  };
+
+  // Memos
+  const pcrOptions = React.useMemo(() => [
+    {
+      value: 'positive',
+      label: t('questionary:resultPcrTest.options.positive'),
+    },
+    {
+      value: 'negative',
+      label: t('questionary:resultPcrTest.options.negative'),
+    },
+  ], [t]);
+
+  return (
+    <MainContainer>
+      <ProgressIndicator
+        currentStep={metadata?.current}
+        totalSteps={metadata?.total}
+        progressBar
+      />
+      <QuestionText extraSpace first>
+        {t('questionary:whenPcrTest')}
+        <QuestionAllApply>{t('questionary:whenPcrTestCaption')}</QuestionAllApply>
+      </QuestionText>
+      <Controller
+        control={control}
+        name="pcrTestDate"
+        defaultValue={undefined}
+        render={({ onChange, value }) => (
+          <DatePicker
+            label="Date"
+            value={value ? new Date(value) : null}
+            locale={i18n.language}
+            onChange={onChange}
+          />
+        )}
+      />
+
+      <QuestionText extraSpace>
+        {t('questionary:resultPcrTest.question')}
+      </QuestionText>
+      <Controller
+        control={control}
+        name="pcrTestResult"
+        defaultValue={undefined}
+        render={({ onChange, value }) => (
+          <OptionList
+            singleSelection
+            value={{ selected: value ? [value] : [] }}
+            onChange={v => onChange(v.selected[0])}
+            items={pcrOptions}
+          />
+        )}
+      />
+
+      {/* Bottom Buttons */}
+      <p><ErrorMessage errors={errors} name="name" /></p>
```

Comment:
This will add an empty space when there isn't errors for "name". Is the idea? Otherwise, use the render prop from ErrorMessage to return it.
Also, there is no "name" field here. Which error you will shown?